packages/modules/viewer-react/src/utilities/index.ts

```diff
@@ -33,7 +33,11 @@ export const isFunction = (item: any) => {
  * @returns
  */
 export const isEqual = (a?: any, b?: any) => {
-  if (!a || !b) {
+  if (a === undefined && b === undefined) {
+    return true;
+  }
+  // Do not check !a or !b as value can be boolean type
+  if (a === undefined || b === undefined) {
```

Comment:
Ah missed this. I _think_ it could be simplified:
`a ?? b` if you can check for null as well as undefined.

Reply:
Actually it'd be `!(a ?? b)`, I think

Reply:
Nah disregard